infra/archiveStorage.bicep

```diff
@@ -0,0 +1,129 @@
+param location string = resourceGroup().location
+
+var tags = {
+  'cost-category': 'core'
+}
+
+@allowed([
+  'Premium_LRS'
+  'Premium_ZRS'
+  'Standard_GRS'
+  'Standard_GZRS'
+  'Standard_LRS'
+  'Standard_RAGRS'
+  'Standard_RAGZRS'
+  'Standard_ZRS'
+])
+param skuName string
+
+var unique = substring(uniqueString(resourceGroup().id), 0, 8)
+
+resource blobStorage 'Microsoft.Storage/storageAccounts@2022-09-01' = {
+  name: 'blobsswwebsite${unique}'
+  location: location
+  tags: tags
+  sku: {
+    name: skuName
+  }
+  kind: 'BlobStorage'
+  properties: {
+    allowBlobPublicAccess: true
+    publicNetworkAccess: 'Enabled'
+    accessTier: 'Hot'
+    supportsHttpsTrafficOnly: true
+  }
+}
+
+resource blobServices 'Microsoft.Storage/storageAccounts/blobServices@2022-09-01' = {
+  name: 'default'
+  parent: blobStorage
+  properties: {
+    changeFeed: {
+      enabled: false
+    }
+    restorePolicy: {
+      enabled: false
+    }
+    containerDeleteRetentionPolicy: {
+      enabled: true
+      days: 7
+    }
+    deleteRetentionPolicy: {
+      allowPermanentDelete: false
+      enabled: true
+      days: 7
+    }
+    cors: {
+      corsRules: [
+        {
+          allowedHeaders: [
+            '*'
+          ]
+          allowedMethods: [
+            'GET'
+            'HEAD'
+            'OPTIONS'
+          ]
+          allowedOrigins: [
+            '*'
```

Comment:
Do we want to scope origins down to our known url formats?

Reply:
Moved to https://github.com/SSWConsulting/SSW.Website/issues/1834